package.json

```diff
@@ -43,9 +43,9 @@
   "_nohoist": "any deps that should not be hoisted list under dependencies, https://github.com/npm/rfcs/issues/287",
   "dependencies": {
     "jasmine": "2.0.1",
-    "lit": "1.0.0",
+    "lit": "2.1.3",
```

Comment:
These dependencies in this file under "dependencies" are not actually used and this is just a hack to prevent npm from hoisting the actual dependencies. Sorry, I know this is confusing and is definitely a hack, but Cory and I couldn't find any other way to do it. But these dependencies are not actually used and need to stay the same (on the earliest version of the package). I'm happy to explain more if you're interested. Any dependabot alert that come from these dependencies are false-positives and should be ignored.

@kevinbuhmann , maybe you can help us find a better solution so we can eliminate this hack?